funtofem/model/function.py

```diff
@@ -252,11 +256,11 @@ def set_timing(self, start: int = None, stop: int = None, averaging: bool = None
         self.averaging = averaging
         return self
 
-    def register_to(self, scenario):
+    def register_to(self, obj):
```

Comment:
Do we want to make this change? I think it might negatively impact readability given the rest of the framework. Maybe we can add a comment saying it's typically for scenarios, but can be used for generic objects?

Reply:
Changed back to scenario like we discussed